src/component.cpp

```diff
@@ -85,6 +87,18 @@ void Component::setSourceComponent(const ImportPtr &imp, const std::string &name
     setImportReference(name);
 }
 
+void Component::appendMath(const std::string &mathString) {
+    mPimpl->mMath.append(mathString);
+}
+
+std::string Component::getMath() const{
+    return mPimpl->mMath;
+}
+
+void Component::removeMath() {
+    mPimpl->mMath = "";
```

Comment:
Similarly to the earlier comment, `mMath.clear()` would perhaps be more natural. This is nit-picking though :)